src/components/Search/SearchFeedback.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './SearchFeedback.scss';
+import throttle from 'lodash/throttle';
+
+import { Icon } from '@patternfly/react-core/dist/dynamic/components/Icon';
+import OutlinedThumbsUpIcon from '@patternfly/react-icons/dist/dynamic/icons/outlined-thumbs-up-icon';
+import OutlinedThumbsDownIcon from '@patternfly/react-icons/dist/dynamic/icons/outlined-thumbs-down-icon';
+import { MenuGroup, MenuItem } from '@patternfly/react-core/dist/dynamic/components/Menu';
+import { useSegment } from '../../analytics/useSegment';
+import type { SearchItem } from './SearchTypes';
+import { SegmentEvent } from '@segment/analytics-next';
+
+export type SearchFeedbackProps = {
+  query: string;
+  results: SearchItem[];
+};
+
+const SearchFeedback = ({ query, results }: SearchFeedbackProps) => {
+  const { ready, analytics } = useSegment();
+  const trackFeedback = (type: string | SegmentEvent) => {
+    ready && analytics && analytics.track(type, { query, results });
+  };
+  const throttledTrackFeedback = throttle(trackFeedback, 5000);
```

Comment:
An alternative to this approach would be to only allow feedback once per query string, and feedback cannot be submitted again until the user types in a new query.

Reply:
@Hyperkid123 agree on both points. I will follow up with another PR. Thanks